src/common/utils.js

```diff
@@ -0,0 +1,26 @@
+import { Console, Random } from '@woowacourse/mission-utils';
+import { LOG_MESSAGE, TAG, GAME_RULES } from './constants.js'; 
+
+const getCarName = () => { return Console.readLineAsync(LOG_MESSAGE.START_MESSAGE); }
+
+const getAttempt = () => { return Console.readLineAsync(LOG_MESSAGE.ATTEMPT_COUNT_MESSAGE); }
+
+const getRandomNumber = (min, max) => { return Random.pickNumberInRange(min, max); };
```

Comment:
```suggestion
const getCarName = () => Console.readLineAsync(LOG_MESSAGE.START_MESSAGE); 

const getAttempt = () => Console.readLineAsync(LOG_MESSAGE.ATTEMPT_COUNT_MESSAGE); 

const getRandomNumber = (min, max) =>  Random.pickNumberInRange(min, max);
```
저는 이렇게 경우에는 중괄호와 `return`문을 생략하는데 다은님은 어떻게 생각하시나요??